cmp/filter/FilterChooserFieldSpec.js

```diff
@@ -77,6 +81,10 @@ export class FilterChooserFieldSpec extends BaseFilterFieldSpec {
         let ret;
         if (isFunction(this.valueRenderer)) {
             ret = this.valueRenderer(value, op);
+            if (isValidElement(ret)) {
+                // Prevents [object Object] rendering
+                ret = renderToStaticMarkup(ret);
+            }
```

Comment:
I was initially concerned that we were going to emit markup somewhere in the stack, given that the rendering pipeline for `FilterChooser` is fully React (as opposed to the issue handled by #3154 where we pipe into ag-grid context menus that require strings).

But that point is rendered moot 😁  by the use of `stripTags` below, which is where we first ended up with [object Object] before this fix.

I thought maybe we could update that to let this method return either a stripped string _or_ an element, but we look to do string-based comparisons downstream so, yeah, we def. need this to return a string